scripts/build-espresso.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const { logger, fs } = require('@appium/support');
+const { ServerBuilder } = require('../build/lib/server-builder.js');
+
+const LOG = new logger.getLogger('EspressoBuild');
+
+const ROOT_DIR = path.resolve(__dirname, '..');
+const ESPRESSO_SERVER_ROOT = path.join(ROOT_DIR, 'espresso-server');
+
+const ESPRESSO_SERVER_BUILD = path.join(ESPRESSO_SERVER_ROOT, 'app', 'build');
+
+async function buildEspressoServer () {
+  console.log(`Deleting the build directory ${ESPRESSO_SERVER_BUILD}`); // eslint-disable-line no-console
+
+  const opts = {
+    serverPath: ESPRESSO_SERVER_ROOT,
+    showGradleLog: process.env.SHOW_GRADLE_LOG ? process.env.SHOW_GRADLE_LOG : false
+  };
+
+  if (process.env.TEST_APP_PACKAGE) {
+    opts.testAppPackage = process.env.TEST_APP_PACKAGE;
+  }
+
+  if (process.env.ESPRESSO_BUILD_CONFIG) {
+    if (!(await fs.exists(process.env.ESPRESSO_BUILD_CONFIG))) {
+      throw Error(`'${process.env.ESPRESSO_BUILD_CONFIG}' did not exist. Please set the path as an absolute path.`);
+    }
+    try {
+      const buildConfigurationStr = await fs.readFile(process.env.ESPRESSO_BUILD_CONFIG, 'utf8');
+      opts.buildConfiguration = JSON.parse(buildConfigurationStr);
+      console.log(`The espresso build config is ${JSON.stringify(opts.buildConfiguration)}`); // eslint-disable-line no-console
+    } catch (e) {
+      throw Error(`Failed to parse the ${process.env.ESPRESSO_BUILD_CONFIG}. Please make sure that the JSON is valid format. Error: ${e}`);
+    }
+  }
+
+  const builder = new ServerBuilder(LOG, opts);
+  await builder.build();
+
+  const dstPath = path.resolve(ESPRESSO_SERVER_ROOT, 'app', 'build', 'outputs', 'apk', 'androidTest', 'debug');
+  console.log(`The built server apk, app-debug-androidTest.apk, is in ${dstPath}`); // eslint-disable-line no-console
```

Comment:
I would rather just show the full path together with the file name

Reply:
I considered the way as well, but I expected the primary usage was to open the path with Finder (on macOS for example) to refer to the apk instead of opening the apk itself.

Some terminal tools can open the path directly. e.g. cmd + click the link. When they open the path, if the path is `.apk`, the system will open an application instead of the Finder. Some tools require selecting the entire path and copy&paste it. This does not have much differences in the full path vs the file's directory.

Thus, at this time, showing the directory helps the expected use case with some tools, but other tools do not have differences to reach the .apk. So I did the current method.

I like the current way for this use case I expected, but I can put the full path instead.